engine/src/main/java/org/terasology/engine/subsystem/headless/renderer/HeadlessWorldRenderer.java

```diff
@@ -71,6 +73,21 @@ public boolean renderLightComponent(LightComponent lightComponent, Vector3f ligh
     }
 
     @Override
+    public void setRenderGraph(RenderGraph renderGraph) {
+
+    }
+
+    @Override
+    public void setShadowMapNode(ShadowMapNode shadowNode) {
```

Comment:
I'm sceptical that this method would be in `WorldRenderer`. What's the rationale behind it?